master/pkg/model/experiment_config.go

```diff
@@ -250,6 +250,19 @@ func (r OptimizationsConfig) Validate() []error {
 	}
 }
 
+// BindMountsConfig is the configuration for bind mounts.
+type BindMountsConfig []BindMount
+
+// UnmarshalJSON implements the json.Unmarshaler interface.
+func (b *BindMountsConfig) UnmarshalJSON(data []byte) error {
+	unmarshaled := make([]BindMount, 0)
```

Comment:
non-blocking: I like `var unmarshaled []BindMount` a little better (something about not distinguishing empty slices from `nil` ones), though to be honest I'm still not sure what would be considered better Go style.

Reply:
Why do you want not to distinguish empty slices from nil ones? Golang doesn't always prevent nil dereference.

Reply:
Oh wait, there actually is a definitive answer on this: https://github.com/golang/go/wiki/CodeReviewComments#declaring-empty-slices. (Well, that distinguishes `var t []T` from `t := []T{}`, but all the points apply the same if the latter is replaced by `make([]T, 0)`. Nil slices can't be dereferenced like nil pointers can, so I don't think dereferencing is relevant here.